mithril.d.ts

```diff
@@ -446,30 +411,25 @@ declare namespace Mithril {
 	*/
 	interface VirtualElement {
 		/**
-		* A key to optionally associate with this element.
-		*/
-		key?: number;
-
-		/**
 		* The tag name of this element.
 		*/
-		tag?: string;
+		tag: string;
 
 		/**
 		* The attributes of this element.
 		*/
-		attrs?: Attributes;
+		attrs: Attributes;
 
 		/**
 		* The children of this element.
 		*/
-		children?: Array<string|VirtualElement|Component<Controller>>;
+		children: Array<Child>;
```

Comment:
Most problems gives this line.
If make children optional only 5 error, instead of 19.